contracts/argent_account.cairo

```diff
@@ -59,3 +80,22 @@ fn already_initialized() {
     assert(ArgentAccount::get_signer() == 1, 'value should be 1');
     ArgentAccount::initialize(10, 20, 0);
 }
+
+#[test]
+#[available_gas(20000)]
+fn erc165_basic_interfaces() {
```

Comment:
I'd split this in 2, to have the registered only on one side and an unregistered on the other side

Reply:
done